xbmc/video/guilib/VideoGUIUtils.cpp

```diff
@@ -424,6 +427,8 @@ void PlayItem(
       }
 
       const auto parentItem = std::make_shared<CFileItem>(parentPath, true);
```

Comment:
It seems that the "true" here is `isFolder`. Since all the calls you're adding are in the "scope" of the fileitem, wouldn't it make sense to move it to the constructor itself?
Likely not because it might be a folder but not a video one?

Reply:
> Likely not because it might be a folder but not a video one?

 Right.